env-defaults

```diff
@@ -193,11 +193,11 @@ ASF_GDAL_SUBSETTER_IMAGE=asfdataservices/gdal-subsetter:latest
 ASF_GDAL_SUBSETTER_IMAGE_PULL_POLICY='IfNotPresent'
 ASF_GDAL_SUBSETTER_PARALLELISM=2
 
-HARMONY_GDAL_IMAGE=harmony/gdal:latest
-HARMONY_GDAL_IMAGE_PULL_POLICY='IfNotPresent'
-HARMONY_GDAL_PARALLELISM=2
+HARMONY_SERVICE_EXAMPLE_IMAGE=harmonyservices/service-example:latest
+HARMONY_SERVICE_EXAMPLE_IMAGE_PULL_POLICY='IfNotPresent'
```

Comment:
Make sure to set this image pull policy variable in Bamboo. In at least SIT and Sandbox you will need to set it to 'Always' if we're using the latest tag. As a cleanup after you've deployed you should delete any HARMONY_GDAL_* variables from Bamboo.

Reply:
I wasn't planning to have our environments pull from DockerHub as part of this work, because that will mess with how we're producing and locking artifacts right now.  Those deployments are all going straight to ECR.  I will need to update variable names and image names, though.